WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -108,7 +110,6 @@ + (NSArray *)routes
                         @"height": @(statusBarSize.height),
                         },
     @"scale": @([FBScreen scale]),
-    @"locked": @([[XCUIDevice sharedDevice] fb_isScreenLocked])
```

Comment:
This entry in the screen object seems still relevant, even if there is a direct way to get the screen locked status.

Reply:
the problem with that is that WDA detects application as not running/crashed if the screen is locked and the endpoint is called. You can check the attached CI log.

Reply:
Ok. If there is a reason for removing it, I'm fine.